store/sqlstore/utils.go

```diff
@@ -52,3 +53,26 @@ func finalizeTransaction(transaction *gorp.Transaction) {
 		mlog.Error("Failed to rollback transaction", mlog.Err(err))
 	}
 }
+
+// removeNonAlphaNumericTerms removes all words that only contains non-alphanumeric chars from given line
+func removeNonAlphaNumericTerms(line string) string {
```

Comment:
I'd include the character separator, for the split, as an argument so this function is more generic

Reply:
Nice suggestion @ethervoid. I have implemented that in [this commit](https://github.com/amarjeetanandsingh/mattermost-server/commit/47556c0719ea24c604cf590ad538e677a1240175).